.github/dependabot.yml

```diff
@@ -0,0 +1,68 @@
+version: 2
+updates:
+- package-ecosystem: npm
```

Comment:
```suggestion
- package-ecosystem: yarn
```

Reply:
Nope, should be npm...
Weird table...
![image](https://user-images.githubusercontent.com/11037542/116588985-09254200-a8d1-11eb-9a00-06e07190ffa9.png)